packages/sources/src/components/lists/lists.models.ts

```diff
@@ -0,0 +1,10 @@
+export enum ListType {
+  Ul = 'ul',
+  Ol = 'ol'
+}
+
+export interface Lists {
+  type?: ListType;
+  listItems: string[];
```

Comment:
Het is niet nodig de naam van de property te herhalen, bij `type` en `modifier` doen we dat ook niet.